pkg/apiserver-gen/.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+7.0.0-SNAPSHOT
```

Comment:
Can you regenerate the server, now that we are using a stable image tag for the generator? I guess the version of the generator should match the version reported here, so we should no longer be using a `SNAPSHOT` version..